src/meta/src/hummock/manager/mod.rs

```diff
@@ -996,7 +995,11 @@ where
             // policy.
             self.compactor_manager
                 .report_compact_task(context_id, compact_task);
-
+            // Tell compaction scheduler to resume compaction if there's any compactor becoming
+            // available.
+            if assigned_task_num == self.compactor_manager.max_concurrent_task_number() {
+                self.try_resume_compaction();
```

Comment:
Need some log to track `try_resume_compaction` and `pause_when_no_idle_compactor`.

Great job ! Rest LGTM